packages/run-python-kit/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "@google-labs/run-python-kit",
+  "publishConfig": {
```

Comment:
Don't need the `publishConfig` if we also switch to the new `@breadboard-ai` NPM namespace.